Orange/widgets/tests/test_widget.py

```diff
@@ -192,6 +192,18 @@ def test_status_bar_action(self):
         w.statusBar().hide()
         self.assertFalse(action.isChecked())
 
+    def test_widgets_cant_be_subclassed(self):
+        # pylint: disable=unused-variable
+        with self.assertWarns(RuntimeWarning):
+            class MySubWidget(MyWidget):
+                pass
+
+        with warnings.catch_warnings():
+            warnings.filterwarnings("error", ".*", RuntimeWarning)
```

Comment:
Why should we catch warnings here? There aren't any. Or have I missed something?